modules/claims_api/config/schemas/v2/2122.json

```diff
@@ -108,14 +111,6 @@
     "claimant": {
       "type": "object",
       "additionalProperties": false,
-      "required": [
```

Comment:
Should there be a description to tell consumers what is conditionally required?

Reply:
Yes, forgot to add that into the schemas, fixing now, modelling it after the 526 language for the same scenario.